CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Changelog
 
+* [CHANGE] Updated the minimum required Go version to 1.20 and update Drone config #420
 * [CHANGE] Updated the minimum required Go version to 1.19 to ensure compatibility with OpenTelemetry version 1.14.0. #350
```

Comment:
I think we can remove the "1.19" update entry now. (Normally we would keep it in previous release, but we are not good at making dskit releases).

Reply:
Yes, I actually meant to _replace_ the Go 1.19 entry with a Go 1.20 one :)

Reply:
Done

Reply:
Thanks!